src/main/resources/assets/immersiverailroading/wiki/en_us/casting-basin.txt

```diff
@@ -9,6 +9,7 @@ It is built from:
 * 126 x Sand
 *  86 x Blast Brick
 *  63 x Steel Scaffolding
+*   1 x Block of Steel
```

Comment:
Please don't update the wiki files directly, I regenerate them from the github wiki before release

Reply:
I right, now it's all coming back. I must have forgotten to delete those changes after I updated the github wiki. Sorry about that.